src/lighter/main.py

```diff
@@ -243,6 +254,9 @@ def verify(filenames, targetdir=None):
     
     deploy_parser.add_argument('filenames', metavar='YMLFILE', nargs='+',
                        help='Service files to expand and deploy')
+    
+    deploy_parser.add_argument('-p', '--password-check-fail', dest='failOnPassword', help='Fail on password/key check triggering [default: %(default)s]',
```

Comment:
Look awesome! Not let's just get some proper secrets mgmt/encryption support before merging :)

Perhaps use more generic term for "secrets" and avoid short-flag because it's then less obvious what's up when it fails, e.g.

```
 +    deploy_parser.add_argument('--verify-secrets', dest='verifySecrets', help='Fail verification if unencrypted secrets are found [default: %(default)s]', 
 +                      action='store_true', default=False) 
```